### ISSUE TITLE
GoBILDA 5 Turn And ServoImplEx

### ISSUE THREAD

**Issue by yoavmlotok** [FTCLib/FTCLib#247]
TL;DR add ServoImplEx to a FTClib servo class

My team recently built a coaxial swerve robot that uses GoBILDA 5 Turn Superspeed servos for rotating the wheels, these servos need signals between 500 and 2500 milliseconds to make advantage of their full range, but the Control Hub and Expansion Hub send signals between 600 and 2400 milliseconds.
This issue can be fixed by using the ServoImplEx class and setting the range using ```setPwmRange()```
Unfortunately FTClib-ServoEx doesn't support this, which means a second ServoImplEx object of the same servo has to be created to apply the fix.

I'd like it if ServoImplEx functionality would be included in the ServoEx interface.

Alternatively a separate class with ServoImplEx functionality could be created, similar to SimpleServo, perhaps something that would be called ExpandedServo or ComplexServo.

There may be other options that I couldn't think of.

Resource for understanding the problem: https://gm0.org/en/latest/docs/software/adv-control-system/sdk-servos.html

**Comment by JIceberg** [comment]
Thanks for the suggestion! I will add this in the next version.
